webshare/share-securecontext.http.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
> `http.html` does not mean anything.

Are you sure? The links above in the comment generated by w3c-bots all link to https://w3c-test.org except the `http.html` file, which links to http://w3c-test.org. It may just be the bot that generates the GitHub comment though, it may not affect the test runner.

> On your local machine, you should be setting web-platform.test and a few other domains to point to loopback

Yeah, I've done that and it works there. But that's a manual process. What I'm interested in is the test runner at https://w3c-test.org/tools/runner/index.html --- ideally it should automatically run the `http.html` test on an `http://` URL so I can run the full battery of tests without having to manually run that test on http. Maddeningly I am unable to test this anywhere:

- On my local machine, I can run http://localhost:8000/tools/runner/index.html. It works great, except the `http.html` test fails because `localhost` is a secure context.
- Still on my local machine, http://web-platform.test:8000 works, but https://web-platform.test:8000 doesn't work (the server prints "Bad HTTP/0.9 request type" / "Bad request syntax" and a bunch of junk, and Chrome says "web-platform.test sent an invalid response"). Despite OpenSSL being installed. So I can't test the behaviour of the test runner switching to `http://` locally.
- https://w3c-test.org/tools/runner/index.html prints "Updating and loading test manifest; this may take several minutes." and it never completes (for any tests), even after an hour. I'm trying it with `/payment-request`, which has a `http.html` file. This was working for me a few days ago but hasn't been working for 24 hours.
- https://w3c-test.org/submissions/6287/tools/runner/index.html (the test runner inside this PR's submission directory) completes for `/webshare` but shows 0 tests. Trying to test it on `/payment-request`, it also runs for what seems like forever.

(In my experience with this tool locally, if the server crashes with a Python error locally, the client hangs forever at the "Updating and loading test manifest" step, so maybe there is a server-side crash.) This test runner has been the bane of my existence for the past few days :/